.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ThirdParty/Bolts/BoltsInClasspath/Bolts-ObjC-1.8.4"]
+	path = ThirdParty/Bolts/BoltsInClasspath/Bolts-ObjC-1.8.4
+	url = https://github.com/BoltsFramework/Bolts-ObjC.git
```

Comment:
This is stored in the database itself. I'm assuming the hash is for Bolts 1.8.4.